tests/unit/services/ember-cli-test.js

```diff
@@ -65,6 +65,21 @@ test("buildProperties() works as expected with replacements", function (assert)
   assert.ok(props.content.indexOf('helper(myHelper)') !== -1, 'Replacements worked if multiple');
 });
 
+test("buildProperties('model') creates a valid file for a model", function(assert) {
```

Comment:
This test is too brittle, breaking with any change to blueprints/model.js

I suggest removing.